app/helpers/application_helper.rb

```diff
@@ -1,5 +1,11 @@
 module ApplicationHelper
+
   def label_helper(form, text, id)
     form.label text, class: 'col-2 col-form-label', :for => id
   end
+
+  def edit?(type)
+    type == 'edit'
+  end
```

Comment:
This method is too general and being frankly it smells :-) Maybe I will later find the words to express what I mean.